app/api/dao/organization.py

```diff
@@ -88,6 +70,89 @@ def update_organization(data):
             return update(organization, data, messages.ORGANIZATION_SUCCESSFULLY_UPDATED, HTTPStatus.OK)     
         return messages.NOT_ORGANIZATION_REPRESENTATIVE, HTTPStatus.FORBIDDEN
 
+    
+    @staticmethod
+    def list_organizations(
+        search_name,
+        page,
+        per_page,
+        token
+    ):
+
+        """ Retrieves a list of organizations with the specified ID.
+        
+        Arguments:
+            user_id: The ID of the user to be listed.
+            search_name: The search query for name of the organizations to be found.
```

Comment:
rename search_name to organization name

Reply:
@meenakshi-dhanani , please see my comment above

Reply:
Ok, I've changed the search_name to just name now, @meenakshi-dhanani . Let me know if there's anything else need modification